tests/test_main.py

```diff
@@ -29,3 +57,15 @@ def test_main(_mock1, _mock2, _mock3, _mock4):
     """Test the function main."""
     resp = main()
     assert (isinstance(resp, dict))
+
+
+@mock.patch('f8a_report.main.ReportHelper', return_value=MockReportHelper)
+@mock.patch('f8a_report.main.manifest_interface', return_value=True)
+@freeze_time("2020-04-06")
+def test_environment(_mock1, _mock2):
+    """Test the Weekday 0, Monday and GENERATE_MANIFESTS functionality."""
+    resp = main()
+    assert datetime.datetime.today().weekday() == 0
+    assert (isinstance(resp, tuple))
+    assert _mock2().retrieve_stack_analyses_content()[0] is True
+    assert os.environ.get('GENERATE_MANIFESTS') in ['true', 'false']
```

Comment:
Do a equality check.

assert os.environ.get('GENERATE_MANIFESTS') == 'true'.

Reply:
It can be true or false..? can't be equality checks

Reply:
It should be true for positive case and false for negative case, but negative test is missing currently.

Reply:
@arajkumar what I mean .. weather "true` or "false" Both values are acceptable and Tests should not  fail for either case.